### PR TITLE
adds helpArticle entity type

### PR DESCRIPTION
**Pull request by nsiskind**

### CODE REVIEW

File: entity_helparticle.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	BaseEntity
+
+	// Admin
+	Keywords *[]string `json:"keywords,omitempty"` // LIST
+
+	// Content
+	Name             *string `json:"name,omitempty"`             // STRING
+	Body             *string `json:"body,omitempty"`             // RICH_TEXT
+	ShortDescription *string `json:"shortDescription,omitempty"` // STRING
+	VoteCount        *int    `json:"voteCount,omitempty"`        // STRING
```
Comment (AidanFitzgerald) [inline entity_helparticle.go:19]: are these actually ints in the JSON? feel like with number fields we usually just use a string but I do see some ints in the hotel entity type. they also use a double pointer in that entity def but i don't think that's necessary?

File: entity_helparticle.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	VoteSum          *int    `json:"voteSum,omitempty"`          // STRING
+	Promoted         **bool  `json:"promoted,omitempty"`         // BOOl
+
+	ExternalArticlePostDate   *Date `json:"externalArticlePostDate,omitempty"`
```
Comment (AidanFitzgerald) [inline entity_helparticle.go:23]: where is Date defined? can't find it lol. For date fields I think we also usually use a *string but could be wrong here

File: entity_helparticle.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	// Knowledge Assistant
+	PrimaryConversationContact *string `json:"primaryConversationContact,omitempty"`
+	NudgeEnabled               **bool  `json:"nudgeEnabled,omitempty"`
+
+	// Internal Use
+	Folder   *string `json:"folder,omitempty"`
+	Timezone *string `json:"timezone,omitempty"`
+}
+
+func (j *HelpArticle) String() string {
```
Comment (AidanFitzgerald) [inline entity_helparticle.go]: prob switch this to h

File: entity_helparticle.go (hunk 2 above)
Comment (nsiskind) [reply entity_helparticle.go:23]: date is in location_customfield.go ... its just a string though. like this

type Date string

File: entity_helparticle.go (hunk 1 above)
Comment (nsiskind) [reply entity_helparticle.go:19]: https://api-sandbox.yext.com/v2/accounts/me/entities/123?api_key=44d8cc0586f38db2503c8203ccf4e0ec&v=20200625

it does come back as an int ... i am not sure whether i should use a double pointer. not sure its necessary... that is when we want to send nil objects right?

File: entity_helparticle.go (hunk 3 above)
Comment (nsiskind) [reply entity_helparticle.go]: 👍

**Review by AidanFitzgerald — APPROVED** [review]

**Pull request merged**